x-pack/plugins/security_solution/common/siem_migrations/model/common.gen.ts

```diff
@@ -20,10 +20,7 @@ import { z } from '@kbn/zod';
  * A string that is not empty and does not contain only whitespace
  */
 export type NonEmptyString = z.infer<typeof NonEmptyString>;
-export const NonEmptyString = z
-  .string()
-  .min(1)
-  .regex(/^(?! *$).+$/);
+export const NonEmptyString = z.string().min(1).regex(/\S+/);
```

Comment:
I was thinking to re-se the common `NonEmptyString` type from `x-pack/plugins/security_solution/common/api/model/primitives.gen.ts`. I will need to use `RuleResponse` in my "prebuilt rules" PR and it imports the `NonEmptyString` from that common folder. So, either we would need to rename our non empty string type (if we want to keep it) or we just re-use the above type. Having cases redefined type error.